cumulus_library/protected_table_builder.py

```diff
@@ -32,22 +37,28 @@ class ProtectedTableBuilder(base_table_builder.BaseTableBuilder):
 
     def prepare_queries(
         self,
-        cursor: databases.DatabaseCursor,
-        schema: str,
-        study_name: str,
-        study_stats: dict,
+        config: base_utils.StudyConfig,
+        manifest: study_manifest.StudyManifest | None = None,
         *args,
-        manifest: study_parser.StudyManifestParser | None = None,
+        study_stats: dict | None = None,
         **kwargs,
     ):
+        if study_stats is None:
+            study_stats = {}
```

Comment:
extremely minor nit: this could be a one-liner with `study_stats = study_stats or {}`